Sync 21-02-2019

mazipan opened the pull request.

mul14 commented:
Terima kasih atas kontribusinya. :)

agasigp commented:
Sama-sama Mas Mul. Ini sambil dicicil juga. Kalau ada update lagi aku kirim PR lagi.

bencodezen committed:
Feature/add new events links (#1993)

* refactor: Swap in new meetup link

* feature: Add Events link

pieer committed:
Vue Mastery Free Weekend announcement (#1997)

In preparation for Vue Mastery Free Weekend announcement on Friday here is a pull request that: 
- Add a banner at the top of the site vue.org
- Redirect user to vue mastery in a new tab on click
- The banner disappear after scroll to only show the static nav and the content
- Let the user remove the banner
- The banner won’t show up again after the user clicked on the close button (saved in local storage)
- Keep the usual mobile behaviour with static header
- Tested on Chrome, Safari, and firefox

Jinjiang committed:
Update components-slots.md (#2003)

NataliaTepluhina committed:
Workaround for code quote (#1979)

phanan committed:
Remove Handlebars reference (fixes 1780) (#1972)

* Remove Handlebars ref (fixes 1780)

* Fix wrong use of vice versa oh my English sucks

atjason committed:
Update onUpdate callback. (#2002)

Commit:
Closes #1992 (#2000)

Workaround for issue with raw tags rendering on certain pages.

Continuation of PR #1979

benhalpern committed:
Add dev.to link to ecosystem dropdown (#1980)

hadalin committed:
Update components-slots.md (#1991)

Fix typo.

jnuricumbo committed:
Fix documentation Debugging in VSCode (#1994)

* Update sourceMapPathOverrides according to https://github.com/Microsoft/VSCode-recipes/tree/master/vuejs-cli

afontcu committed:
Add negative example warning (#1990)

robinchow committed:
Updated el and $el from HTMLElement to Element (#1996)

As per this merged PR: https://github.com/vuejs/vue/pull/8809

NataliaTepluhina committed:
Clarified v-model props/events for native elements (#1974)

* Added details for v-model

Explained what events and properties v-model  for different input elements

* fix: refactored tip to bullet points

phanan committed:
More fixes for %raw% tags showing (#2005)

afontcu committed:
Clarify null and undefined values on typed props (#1937)

chadidi committed:
[arb-mod] add Arabic subtitles for "why vue" video (#1808)

* Translate 13 seconds of the presentation video to arabic

* Translated with google translator and need review from message 17 and up

* first review from 1 to 16

* Update assets/why-vue/arabic.js.srt

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 45

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 46

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update translation from line 1 to 51

* Update line 79

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update translation from line 52 to 102

* Update line 114

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update translation from line 103 to 163

* Update line 171

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 211

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update translation from line 164 to 266

* Update line 274

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 294

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 298

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 314

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 290

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 342

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 374

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update line 402

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update translation from line 267 to 418

* Update line 462

Co-Authored-By: chadidi <chadidi97@gmail.com>

* Update translation from line 419 to 587

btjk committed:
Fix typo in new slots shorthand syntax (#1987)

MachinisteWeb committed:
Indent fix (Biggest update ever) (#1893)

Signed-off-by: Bruno J. S. Lesieur <bruno.lesieur@gmail.com>

MSCAU committed:
Added usage information about inline-templates and x-templates (#1853)

* Added usage information about inline-templates and x-templates

I wasted a whole day on this issue as it wasn't documented, AFAIK. StackOverflow people thankfully were able to help: https://stackoverflow.com/questions/52940350/vue-component-error-property-or-method-x-is-not-defined-on-the-instance-but-ref

* Corrected inline-template sentence

Found that the inline-template syntax requires the component template definition to reside within the DOM element to which Vue is attached.

Jinjiang committed:
Update components-edge-cases.md (#2010)

NataliaTepluhina committed:
fix: replaced prop 'data' in the fiddle with 'heroes' (#2012)

vue-bot committed:
chore: update sponsors (#2017)

vue-bot committed:
chore: update sponsors (#2018)

bencodezen committed:
chore: Update info

NataliaTepluhina committed:
Merge pull request #2020 from bencodezen/update-bens-info

chore: Update Ben Hong info

tauantcamargo committed:
It solves modal error in home page (#2021)
